packages/common-integration-tests/package.json

```diff
@@ -47,6 +47,7 @@
   },
   "dependencies": {
     "@gomomento/sdk-core": "file:../core",
+    "@gomomento/generated-types": "0.61.0",
```

Comment:
I don't think we should need this dependency - this seems like it is indicating that we have leaked something from one of the gRPC APIs into the tests.